Cargo.toml

```diff
@@ -11,14 +11,11 @@ getset = "0.1"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 once_cell = "1.4"
-kv = "0.20"
 iota-core = { git = "https://github.com/iotaledger/iota.rs", branch = "dev" }
 url = { version = "2.1", features = [ "serde" ] }
 tokio = "0.2"
 rand = "0.3"
-
-# sql
-rusqlite = { version = "0.23", optional = true }
+rusqlite = "0.23"
```

Comment:
What do you think about some other choice like `sqlx`? It can also support different databases.

Reply:
to use `sqlx` here I'd need to change the StorageAdapter to make every method `async`, but I like the idea and I'll do it on a separate PR.